### PR TITLE
update to contributor guide

### PR DESCRIPTION
**Pull request by sdwheeler**
Updated contributor guide

Issue #1106 fix styleguide to reflect platyps schema

- Add note about not splitting code samples with paragraphs in examples section
- Also not code blocks in any other sections

Issue #15 Style guide

- Update new CommonMark requirements
- Link to Microsoft Style Guide

Update Readme.md

- Doc the structure of the repo

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CONTRIBUTING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/CONTRIBUTING.md) | :white_check_mark:Succeeded | |
[README.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/README.md) | :white_check_mark:Succeeded | |
[contributing/1-GET-STARTED.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/1-GET-STARTED.md) | :white_check_mark:Succeeded | |
[contributing/2-WRITING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/2-WRITING.md) | :white_check_mark:Succeeded | |
[contributing/3-STYLE-GUIDE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/3-STYLE-GUIDE.md) | :white_check_mark:Succeeded | |
[contributing/4-MARKDOWN-SPECIFICS.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/4-MARKDOWN-SPECIFICS.md) | :white_check_mark:Succeeded | |
[contributing/5-FORMATTING-CODE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/5-FORMATTING-CODE.md) | :white_check_mark:Succeeded | |
[contributing/6-UPDATING-REFERENCE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/6-UPDATING-REFERENCE.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C4%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806042309432540-2500%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=m1YbVsP9AiDP%2FMy45iVqY%2BY5lMJqbbnbeUBzJIJy4ZI%3D&st=2018-06-04T23%3A13%3A50Z&se=2018-07-05T23%3A18%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: contributing/4-MARKDOWN-SPECIFICS.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  - Sub-bullet item
+
+    Sentence explaining the sub-bullet.
+
+- Second bullet item
+- Third bullet item
+
+```
+
+This is a list that contain sub-elements under a bullet item.
```
Comment (DCtheGeek) [inline contributing/4-MARKDOWN-SPECIFICS.md]: Contains should be plural.

File: contributing/4-MARKDOWN-SPECIFICS.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+> [!WARNING]
+> This is a warning.
+
+> [!TIP]
+> This is a tip.
+
+> [!IMPORTANT]
+> This is important.
+
```
Comment (DCtheGeek) [inline contributing/4-MARKDOWN-SPECIFICS.md]: Remove additional blank line in code-fence.

File: contributing/3-STYLE-GUIDE.md [hunk 3]
```diff
@@ -175,34 +135,8 @@ Here is the live link:
...
-
-And it will be rendered like this:
-
-![alert boxes](./images/alert-boxes.png)
-
-## Next steps
-
-See [Formatting code blocks](FORMATTING-CODE.md).
+## Next Steps
+See [Markdown Specifics](4-MARKDOWN-SPECIFICS.md)
```
Comment (DCtheGeek) [inline contributing/3-STYLE-GUIDE.md]: Put a blank line between header and content.  Also add a period at the end of the line with the link.

File: contributing/6-UPDATING-REFERENCE.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# Updating reference articles
+
+Cmdlet reference articles have a specific structure. This structure is defined by [PlatyPS].
+PlatyPS is the tool we use to generate the cmdlet help for a PowerShell module and to create the
+MAML help files used by the `Get-Help` cmdlet.
+
+PlatyPS has a hardcoded schema for cmdlet reference that is written into the code. The
+[platyPS.schema.md] document attempts to describe this structure. When updating cmdlet reference,
+if you don't follow this schema you can cause a build errors that must be fixed before we can
```
Comment (DCtheGeek) [inline contributing/6-UPDATING-REFERENCE.md]: Should be 'can cause build errors'

File: contributing/6-UPDATING-REFERENCE.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+accept your contribution.
+
+## Guidelines
+
+- Do not remove any of the header structures. PlatyPS expect a specific set of headers.
+- The **Input type** and **Output type** headers must have a type.
+  If the cmdlet does not take input or return a value then use the value "None".
+- Fenced code blocks are only allows in the **Examples** section.
```
Comment (DCtheGeek) [inline contributing/6-UPDATING-REFERENCE.md]: Should be 'only allowed'

File: contributing/6-UPDATING-REFERENCE.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+The **Examples** header is an H2 header. Each example must be an H3 header. Number the examples and
+add a title for each example. For example:
+
+### Example 1: Get cmdlets, functions, and aliases
+
+```powershell
+Get-Command
+```
+
+This command gets the Windows PowerShell cmdlets, functions, and aliases that are installed on the
```
Comment (DCtheGeek) [inline contributing/6-UPDATING-REFERENCE.md]: Remove the word 'Windows'.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -4,7 +4,9 @@ Thank you for your interest in contributing to quality documentations.
...
-2. [Writing PowerShell documentation](./contributing/WRITING.md)
-3. [Style Guide](./contributing/STYLE.md)
-4. [Formatting Code blocks](./contributing/FORMATTING-CODE.md)
\ No newline at end of file
+1. [Get started](./contributing/1-GET-STARTED.md)
+2. [Writing PowerShell documentation](./contributing/2-WRITING.md)
+3. [Style Guide](./contributing/3-STYLE-GUIDE.md)
+4. [Markdown Specifics](./contributing/4-MARKDOWN-SPECIFICS.md)
+5. [Formatting Code blocks](./contributing/5-FORMATTING-CODE.md)
+6. [Updating Reference](6-UPDATING-REFERENCE.md)
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply CONTRIBUTING.md]: Fixed

File: contributing/3-STYLE-GUIDE.md [hunk 8]
```diff
@@ -1,5 +1,11 @@
 # Style guide for PowerShell-Docs
 
+This articles some style guidance specific to the PowerShell-Docs content.
+There is some limited guidance on specific Markdown formatting.
+Markdown specifics are covered in the [Next steps](#next-steps).
+
+For other writing style guidance, see the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/).
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply contributing/3-STYLE-GUIDE.md]: fixed

File: contributing/4-MARKDOWN-SPECIFICS.md (hunk 2 above)
Comment (sdwheeler) [reply contributing/4-MARKDOWN-SPECIFICS.md]: fixed

File: contributing/3-STYLE-GUIDE.md (hunk 3 above)
Comment (sdwheeler) [reply contributing/3-STYLE-GUIDE.md]: fixed

File: contributing/6-UPDATING-REFERENCE.md (hunk 4 above)
Comment (sdwheeler) [reply contributing/6-UPDATING-REFERENCE.md]: fixed

File: contributing/6-UPDATING-REFERENCE.md (hunk 5 above)
Comment (sdwheeler) [reply contributing/6-UPDATING-REFERENCE.md]: Fixed

File: contributing/6-UPDATING-REFERENCE.md (hunk 6 above)
Comment (sdwheeler) [reply contributing/6-UPDATING-REFERENCE.md]: Fixed

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CONTRIBUTING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/CONTRIBUTING.md) | :white_check_mark:Succeeded | |
[README.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/README.md) | :white_check_mark:Succeeded | |
[contributing/1-GET-STARTED.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/1-GET-STARTED.md) | :white_check_mark:Succeeded | |
[contributing/2-WRITING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/2-WRITING.md) | :white_check_mark:Succeeded | |
[contributing/3-STYLE-GUIDE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/3-STYLE-GUIDE.md) | :white_check_mark:Succeeded | |
[contributing/4-MARKDOWN-SPECIFICS.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/4-MARKDOWN-SPECIFICS.md) | :white_check_mark:Succeeded | |
[contributing/5-FORMATTING-CODE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/5-FORMATTING-CODE.md) | :white_check_mark:Succeeded | |
[contributing/6-UPDATING-REFERENCE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/6-UPDATING-REFERENCE.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C5%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806051450086199-2500%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1iyybsMJAg%2BeP7O540EjZw3JGWAz98JjElRRyVXD2s0%3D&st=2018-06-05T14%3A54%3A28Z&se=2018-07-06T14%3A59%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: contributing/4-MARKDOWN-SPECIFICS.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+  Sentence explaining the first bullet.
+
+  - Sub-bullet item
+
+    Sentence explaining the sub-bullet.
+
+- Second bullet item
+- Third bullet item
+
```
Comment (DCtheGeek) [inline contributing/4-MARKDOWN-SPECIFICS.md]: Remove extra blank line inside code-fence.

File: contributing/6-UPDATING-REFERENCE.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+The **Examples** header is an H2 header. Each example must be an H3 header. Number the examples and
+add a title for each example. For example:
+
+### Example 1: Get cmdlets, functions, and aliases
+
+```powershell
+Get-Command
+```
+
+This command gets the  PowerShell cmdlets, functions, and aliases that are installed on the
```
Comment (DCtheGeek) [inline contributing/6-UPDATING-REFERENCE.md]: Remove extra space in front of PowerShell.

File: contributing/4-MARKDOWN-SPECIFICS.md (hunk 9 above)
Comment (sdwheeler) [reply contributing/4-MARKDOWN-SPECIFICS.md]: Done

File: contributing/6-UPDATING-REFERENCE.md (hunk 10 above)
Comment (sdwheeler) [reply contributing/6-UPDATING-REFERENCE.md]: Argh... done

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CONTRIBUTING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/CONTRIBUTING.md) | :white_check_mark:Succeeded | |
[README.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/README.md) | :white_check_mark:Succeeded | |
[contributing/1-GET-STARTED.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/1-GET-STARTED.md) | :white_check_mark:Succeeded | |
[contributing/2-WRITING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/2-WRITING.md) | :white_check_mark:Succeeded | |
[contributing/3-STYLE-GUIDE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/3-STYLE-GUIDE.md) | :white_check_mark:Succeeded | |
[contributing/4-MARKDOWN-SPECIFICS.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/4-MARKDOWN-SPECIFICS.md) | :white_check_mark:Succeeded | |
[contributing/5-FORMATTING-CODE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/5-FORMATTING-CODE.md) | :white_check_mark:Succeeded | |
[contributing/6-UPDATING-REFERENCE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-styleupdate/contributing/6-UPDATING-REFERENCE.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C5%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806051638140039-2500%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1Yb9y6CwF2opem5Bw73hUIrEy7kteDCGm5AEqxf7R18%3D&st=2018-06-05T16%3A42%3A55Z&se=2018-07-06T16%3A47%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**